Bayesian_Cognitive_Modeling/ParameterEstimation/DataAnalysis/Kappa_Stan.R

```diff
@@ -74,7 +20,7 @@ parameters <- c("kappa", "xi", "psi", "alpha", "beta", "gamma", "pi")
 
 # The following command calls Stan with specific options.
 # For a detailed description type "?rstan".
-samples <- stan(model_code=model,   
+samples <- stan(file="Kappa.stan",   
```

Comment:
I would strongly recommend lowercasing the models and renaming to something informative, like `agreement.stan`.